rskj-core/src/main/java/co/rsk/peg/ErpRedeemScriptBuilderUtils.java

```diff
@@ -0,0 +1,44 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.VerificationException;
+import co.rsk.bitcoinj.script.Script;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static co.rsk.peg.ErpRedeemScriptBuilderCreationException.Reason.INVALID_CSV_VALUE;
+
+public class ErpRedeemScriptBuilderUtils {
+    private static final long MAX_CSV_VALUE = 65535L; // 2^16 - 1, since bitcoin will interpret up to 16 bits as the CSV value
+    private static final Logger logger = LoggerFactory.getLogger(ErpRedeemScriptBuilderUtils.class);
+
+    private ErpRedeemScriptBuilderUtils() {
+    }
+
+    public static void validateRedeemScriptValues(
+        Script defaultFederationRedeemScript,
+        Script erpFederationRedeemScript,
+        Long csvValue
+    ) {
+        if (!defaultFederationRedeemScript.isSentToMultiSig() || !erpFederationRedeemScript.isSentToMultiSig()) {
+
+            String message = "Provided redeem scripts have an invalid structure, not standard";
+            logger.debug(
+                "[validateRedeemScriptValues] {}. Default script {}. Emergency script {}",
+                message,
+                defaultFederationRedeemScript,
+                erpFederationRedeemScript
+            );
+            throw new VerificationException(message);
```

Comment:
I'd avoid throwing this exception here because it's part of bitcoinj-thin, let's use a different type